app/views/owners/_owners_table.html.erb

```diff
@@ -49,6 +55,11 @@
           method: "delete",
           data: { confirm: t("owners.index.confirm_remove") },
           class: "form__submit form__submit--small" %>
+        <br/>
+        <%= button_to "Edit",
```

Comment:
should we hide this on the current user's row, since they are not allowed to update their own role?

Reply:
I don't think we should hide it, but we can disable it so the user cannot proceed further.